src/cljs/nr/cardbrowser.cljs

```diff
@@ -221,15 +275,20 @@
       "Runner" (conj runner-factions "Neutral")
       "Corp" corp-factions)))
 
-(defn filter-alt-art-cards [cards]
-  (let [alt-arts (:alt-arts @app-state)]
-    (filter #(contains? alt-arts (:code %)) cards)))
+(defn- filter-alt-art-cards [cards]
+  (let [alt-arts (:alt-cards @app-state)]
+    (filter #(or (contains? alt-arts (:code %))
+                 (contains? % :future-version)
+                 (contains? % :previous-versions))
+            cards)))
 
-(defn filter-alt-art-set [setname cards]
+(defn- filter-alt-art-set [setname cards]
   (when-let [alt-key (alt-version-from-string setname)]
+    (if (= alt-key :prev)
+      (filter #(or (contains? % :future-version) (contains? % :previous-versions)) cards)
     (let [sa (map first
```

Comment:
think spacing got messed up here, cuz the `let` block doesn't look like the else branch of the `if`